NEWS

```diff
@@ -95,6 +95,7 @@
 * Fixed potential deadlock with cuda_copy and RTR protocol
 * Fixed tag_recv return value on immediate completion  
 * Fixed memory corruption by proper memh handling in tag offload rendezvous
+* Fixed default huge pages allocation 
```

Comment:
`Don't allocate huge pages by default`

Reply:
Changed default allocator to not use reserved huge pages